### PR TITLE
Docs review (Part 2)

### PR DESCRIPTION
**Pull request by HGSilveri**
- Goes over the tutorials (except for QEK)
- Reviews all the additions to the API docs

### CODE REVIEW

File: tutorials/quantum_simulation/Shadow estimation for VQS.ipynb [hunk 1]
```diff
@@ -1,20 +1,5 @@
 {
```
Comment (Louis-PaulHenry) [inline tutorials/quantum_simulation/Shadow estimation for VQS.ipynb:1]: <p>The <code>\ket</code>don't render on the preview. To be checked on the final render.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (Louis-PaulHenry) [inline tutorials/quantum_simulation/Shadow estimation for VQS.ipynb:1]: <p>same as above with the <code>ket</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (Louis-PaulHenry) [inline tutorials/quantum_simulation/Shadow estimation for VQS.ipynb:1]: <p>add plt.show() to get rid of the <code>&lt;matplotlib.lines.Line2D at 0x7f79857e1d30&gt;</code>  line?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>

Comment (Louis-PaulHenry) [inline tutorials/quantum_simulation/Shadow estimation for VQS.ipynb:1]: <p>again with plt.show() ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>

Comment (Louis-PaulHenry) [inline tutorials/quantum_simulation/Shadow estimation for VQS.ipynb:1]: <p>plt.show() ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='80'/>

File: tutorials/quantum_simulation/Spin chain of 3 atoms in XY mode.ipynb [hunk 2]
```diff
@@ -34,7 +34,7 @@
   },
```
Comment (Louis-PaulHenry) [inline tutorials/quantum_simulation/Spin chain of 3 atoms in XY mode.ipynb:34]: <p>The oscillations do not appear in the newer version?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (Louis-PaulHenry) [inline tutorials/quantum_simulation/Spin chain of 3 atoms in XY mode.ipynb:34]: <p>Again the plots do not show up in  the render</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (Louis-PaulHenry) [inline tutorials/quantum_simulation/Spin chain of 3 atoms in XY mode.ipynb:34]: <p>There is a <code>H_{XY} = \frac{1}{2}\sum_{i</code>  right in the middle of the text</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (Louis-PaulHenry) [inline tutorials/quantum_simulation/Spin chain of 3 atoms in XY mode.ipynb:34]: <p>It seems that the outputs are missing in this notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

**Comment by HGSilveri** [comment]
@Louis-PaulHenry Thanks for the review! The missing outputs are on purpose, the notebook is executed when its built. I usually only keep the output for the ones that take a long time to execute. As for the rendering, I have to check those in detail indeed. I'm having some formatting bug that is hard to catch because it doesn't appear locally, only on the readthedocs build

**Comment by HGSilveri** [comment]
Finally fixed...

**Pull request merged**